dist/js/getstream.js

```diff
@@ -3942,6 +3942,12 @@ var StreamClient = /*#__PURE__*/function () {
         if (options.reactions.recent != null) {
           options.withRecentReactions = options.reactions.recent;
         }
+        if (options.reactions.ranking_vars != null) {
+          options.rankingVars = options.reactions.ranking_vars;
+        }
+        if (options.reactions.score_vars != null) {
+          options.withScoreVars = options.reactions.score_vars;
```

Comment:
should we use `options.reactions.with_score_vars;` instead of `options.reactions.score_vars;`